src/scss/tooltip.scss

```diff
@@ -13,6 +13,7 @@
 		cursor: default;
 		transition: opacity 0.25s;
 	}
+
 	.#{$classname}-tooltip__text {
 		border-radius: 3px;
 		padding: 8px 20px;
```

Comment:
On line 21 (not shown in diff):

```scss
font-family: oFontsGetFontFamilyWithFallbacks(BentonSans);
```

1. This is the only call to oFonts: avoid requiring modules for only 1 function
2. Modules are supposed to inherit the font family from their context. Should this line even be kept in v1?